src/views/studio/MyUploadsView/AssetsGroupUploadBar/AssetsGroupUploadBar.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState, useRef } from 'react'
+import {
+  Container,
+  AssetsGroupUploadBarContainer,
+  ProgressBar,
+  Thumbnail,
+  AssetsInfoContainer,
+  UploadInfoContainer,
+  AssetsDrawerContainer,
+} from './AssetsGroupUploadBar.style'
+import { AssetLine } from '../AssetLine'
+import { Icon, Text } from '@/shared/components'
+import { DrawerButton } from '@/components/Topbar/StudioTopbar/StudioTopbar.style'
```

Comment:
Which button is that? It's the `expand` one? If we need it here, let's elevate it to `shared`, we shouldn't rely on another view